src/navigators/IFrameWindow.ts

```diff
@@ -29,19 +29,24 @@ export class IFrameWindow extends AbstractChildWindow {
         super();
         this._timeoutInSeconds = silentRequestTimeoutInSeconds;
 
-        this._frame = window.document.createElement("iframe");
+        this._frame = IFrameWindow.createHiddenIframe();
         this._window = this._frame.contentWindow;
+    }
+
+    private static createHiddenIframe(): HTMLIFrameElement {
```

Comment:
Do you think it would be a good idea to make this public and re-use it in `CheckSessionIFrame`? I haven't been able to figure out a good way to reduce the code that is duplicated over there.

Reply:
Yes it might makes sense to re-use. I do not plan to touch `CheckSessionIFrame` any time soon. Lets merge this as is, we can pretty easily make this function later public, when there is an actual need...